docs/src/modules/spring/pages/actions-publishing-subscribing.adoc

```diff
@@ -1 +1,100 @@
-= Publishing and Subscribing
+= Publishing and Subscribing with Actions
+
+include::ROOT:partial$include.adoc[]
+
+A very common use case when building Microservices is to publish and subscribe to a stream of events. The source of events can be the journal of an event sourced entity, the value entity state changes, a https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] or Apache Kafka topic for asynchronous messaging between services.
+
+With Actions you can:
+
+- subscribe to events emitted by an event sourced entity within the same service.
+- subscribe to state changes emitted by a value entity within the same service.
+- subscribe to events from Event Sourced Entities published as xref:service-to-service.adoc[service to service eventing]
+- subscribe to external events from https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] or https://kafka.apache.org/[Apache Kafka].
+- publish events to a Google Cloud Pub/Sub or Apache Kafka topic.
+
+Messages are guaranteed to be delivered at least once. This means that receivers must be able to handle duplicate messages.
```

Comment:
I think all this section is a good candidate to have in a partial shared between java and spring docs.. It's very self-contained and can't see it change a lot.

small independent note: since we link to google pub/sub, should we add a link to kafka too on the first paragraph?

Reply:
It makes sense. Do you know if we other partial common to Java and Spring already?

Reply:
I'm having second thoughts. I prefer if we leave this optimization after we have the docs in better shape.

Reply:
:) I did add the link in a paragraph below. I move it from there to here.

Reply:
> Do you know if we have other partials common to Java and Spring already?

Yeah, I did it for the intro in eventsourced entities and I think I will do the same for timers. But we can evaluate later if we think it's worth or not.